### PR TITLE
Skip compatibility check if in node

### LINKED ISSUE #32: React Server Rendering Errors
I'm trying to import this library into a React component which gets server rendered first then registers client side after. So essentially i need the same React component to work in the browser and in Node. This is causing issues in Node because i get `Error: Sorry, the Vimeo Player API is not available in this browser.`.
### Expected Behavior

The Javascript API should allow Node as a browser for some behavior. Even if the API does not support many functions in Node, the API should still allow you to at least import it without errors.
### Actual Behavior

When trying to import this library server side in Node I get `Error: Sorry, the Vimeo Player API is not available in this browser.`
### Steps to Reproduce

Import in any file server side.

### PR DESCRIPTION
**Pull request by luwes**
Fixes the compatibility error thrown on node server side rendering.

Fixes #32

### CODE REVIEW

File: src/lib/functions.js [hunk 1]
```diff
@@ -3,6 +3,14 @@
  */
 
 /**
+ * Check to see this is a node environment.
+ * @type {Boolean}
+ */
+/* global global */
+export const isNode = typeof global !== 'undefined' &&
```
Comment (cfebs) [inline src/lib/functions.js]: Is this an accurate check?

Could we instead get away with something dead simple like: `document && document.querySelector`

Comment (luwes) [inline src/lib/functions.js]: I will test this out @cfebs , not 100% sure.

**Comment by EnrikoLabriko** [comment]
Hello

Any news here? @luwes, I can finish this PR if you want because issue (https://github.com/vimeo/player.js/issues/32) is actual for me as well

File: src/lib/functions.js (hunk 1 above)
Comment (luwes) [reply src/lib/functions.js]: @cfebs that test didn't seem to work, not sure why.
Could you approve if it looks alright to you?

**Pull request merged**